src/components/icon-button/icon-button.jsx

```diff
@@ -10,7 +10,14 @@ const IconButton = forwardRef(function IconButton(props, ref) {
   const height = parseInt(viewBox[3], 10);
   const tabIndex = props.active ? 0 : -1;
 
+  function onClick(event) {
+    if (props.disabled) return;
+    event.preventDefault();
+    props.onPress(event);
```

Comment:
I know this wasn't added here, but I'm wondering why we call refer to `click` as `onPress`? 

I looked it up once before and read something about `press` being more-or-less the same but having some mobile-specific distinctions, but if we're not actually using an `onPress` event then this prop is just misleading